sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredential.java

```diff
@@ -35,5 +39,20 @@
      */
     DefaultAzureCredential(ArrayDeque<TokenCredential> tokenCredentials) {
         super(tokenCredentials);
+        this.tokenCredentials = new ArrayList<TokenCredential>(tokenCredentials.size());
+        this.tokenCredentials.addAll(tokenCredentials);
```

Comment:
ArrayList has a ctor that allows you to pass a collection directly.
```suggestion
        this.tokenCredentials = new ArrayList<TokenCredential>(tokenCredentials);
```

Reply:
done.